Sources/Rendering/Core/ColorTransferFunction/index.js

```diff
@@ -494,6 +494,25 @@ function vtkColorTransferFunction(publicAPI, model) {
         x = 0.5 * (xStart + xEnd);
       }
 
+      if (model.discretize) {
```

Comment:
Please add a comment to explain the idea. E.g. 
Linearly map x to [0, numberOfValues-1], round down to the closest integer (i.e. discretize) and map back to mappingRange.